package.json

```diff
@@ -107,6 +106,7 @@
     "@changesets/cli": "^2.14.0",
     "@octokit/rest": "^19.0.3",
     "@playwright/test": "^1.32.3",
+    "@scaffolder/cli": "0.1.0-next.0",
```

Comment:
Internal dependencies use "workspace:^" ranges, not exact versions. But do we need a dependency here at all?

Also for discussion among the maintainers - should it really be named like this? I know we have `@techdocs/cli`, but that seems wrong to me. Would have preferred `@backstage/techdocs-cli` or `@backstage/plugin-techdocs-cli` if anything

Reply:
Will change it to what is preferred by all of you :) agree that it makes sense.